### PR TITLE
Rating & toast components

### PR DESCRIPTION
**Pull request by kshitijthatte**
Rating components 
 - Filled Ratings
 - Rating Badge

Toast components
- Standard Toast
- Centered Toast
- Stacked Toast

### CODE REVIEW

**Review by manojasarsa — APPROVED** [review]
Good to go 👍

File: components/toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+.toast {
+  position: fixed;
+  left: 1rem;
+  bottom: 1rem;
+  background-color: var(--white);
+  display: flex;
+  align-items: center;
+  gap: 2rem;
+  border-radius: 5px;
+  box-shadow: rgb(0 0 0 / 12%) 0 0 5px;
```
Comment (mimansha-swarup) [inline components/toast/toast.css:10]: hey you can use variable for box shadows too
btw what is this rgb(0 0 0 / 12%) I am curious 😅 
alpha value?

**Review by mimansha-swarup — APPROVED** [review]
seems clean 👍

**Review by BelideNikhil — APPROVED** [review]
Clean code!
Good job!

File: components/toast/toast.css (hunk 1 above)
Comment (kshitijthatte) [reply components/toast/toast.css:10]: Thanks for the suggestion, will add variable while refactoring. Yes, that's the alpha value for opacity.

**Pull request merged**